app/src/renderer/vuex/getters.js

```diff
@@ -23,9 +23,15 @@ export const allTransactions = state =>
 export const wallet = state => state.wallet
 
 // staking
+export const liquidAtoms = state =>
+  (
+    state.wallet.balances.find(
```

Comment:
can we change `wallet.balances` to be an object to avoid using `.find` ?

Reply:
https://github.com/cosmos/voyager/issues/1708

Reply:
ok nice, I labeled it as `high priority`

Reply:
hahaha, ok we should discuss what `high priority` is :D